public/js/services.js

```diff
@@ -293,4 +292,42 @@ angular.module('starter.services', [])
       supplytypes: ['food', 'water', 'shelter', 'blood', 'medical supplies', 'medical services', 'volunteer', 'transportation', 'other contacts']
     };
 
-  });
+}
+
+/* @ngInject */
+function DistrictSelectService(api, ConstEvents, $rootScope) {
```

Comment:
This service deals with keeping state of the current district selection. Since factories are singleton, we can use it to store state of the application instead of rootScope and use getters and setters instead.